app/players/elves/druid.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class Druid(Elf):
+    def __init__(self, favourite_spell: str, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        """Get player info."""
+        return (f"Druid {self.nickname}. {self.nickname} "
```

Comment:
Code style issue: Use double quotes for consistency in your code. Change the single quotes in the f-string to double quotes.